### ISSUE TITLE
bug!???

### ISSUE THREAD

**Issue by huitc** [EnableSecurity/wafw00f#70]
```
$ python3 main.py "http://www.test.com.us/testtest/testtest/testtest/getAtt.php?cat=Hotels&atts=&lang=C"

                                 ^     ^
        _   __  _   ____ _   __  _    _   ____
       ///7/ /.' \ / __////7/ /,' \ ,' \ / __/
      | V V // o // _/ | V V // 0 // 0 // _/
      |_n_,'/_n_//_/   |_n_,' \_,' \_,'/_/
                                <
                                ...'

    WAFW00F - Web Application Firewall Detection Tool

    By Sandro Gauci && Wendel G. Henrique

Checking http://www.test.com.us/testtest/testtest/testtest/getAtt.php?cat=Hotels&atts=&lang=C
Traceback (most recent call last):
  File "main.py", line 392, in <module>
    main()
  File "main.py", line 370, in main
    waf = attacker.identwaf(options.findall)
  File "main.py", line 269, in identwaf
    if self.wafdetections[wafvendor](self):
  File "/data/data/com.termux/files/usr/lib/python3.7/site-packages/wafw00f-0.9.6-py3.7.egg/wafw00f/plugins/barracuda.py", line 17, in is_waf
    r = attack(self)
  File "main.py", line 120, in xssstandard
    return self.request(path=xssstringa, usecache=usecache, cacheresponse=cacheresponse)
  File "/data/data/com.termux/files/usr/lib/python3.7/site-packages/wafw00f-0.9.6-py3.7.egg/wafw00f/lib/evillib.py", line 320, in request
    r = self._request(method, path, headers)
  File "/data/data/com.termux/files/usr/lib/python3.7/site-packages/wafw00f-0.9.6-py3.7.egg/wafw00f/lib/evillib.py", line 378, in _request
    responsebody = response.read()
  File "/data/data/com.termux/files/usr/lib/python3.7/http/client.py", line 460, in read
    s = self._safe_read(self.length)
  File "/data/data/com.termux/files/usr/lib/python3.7/http/client.py", line 612, in _safe_read
    raise IncompleteRead(b''.join(s), amt)
http.client.IncompleteRead: IncompleteRead(157 bytes read, 383 more expected)
$
```

**Comment by 0xInfection** [comment]
This is not a bug. Please use a standard Python interpreter for running wafw00f.

**Comment by sandrogauci** [comment]
@huitc can you provide further information please?

It seems that you are running wafw00f on top of python3 on Android? Also the target site given doesn't appear to resolve so I cannot reproduce your issue. When I test other sites, I don't come across this problem. However it does appear to be a matter of your target site returning less data, triggering an unhandled exception.

**Comment by sandrogauci** [comment]
triaged

**Comment by 0xInfection** [comment]
Will be fixed via #67.

**Comment by sandrogauci** [comment]
@huitc issue is fixed in the latest thanks to updates by @0xInfection
